2.3-tapscript.ipynb

```diff
@@ -758,15 +657,15 @@
     "signature0 =  # TODO: implement\n",
```

Comment:
On line https://github.com/bitcoinops/taproot-workshop/pull/125/files#diff-7ff84fb54ffe0773f137d34707fa3c36R650 

I think output should instead be tx.vout[0]

Reply:
fixed